.yamato/_templates.yml

```diff
@@ -8,29 +8,33 @@
 
 ---
 
-pack_hdrp_template:
-    name: Pack HDRP Template
+pre_pack_hdrp_template_Win_trunk:
+    name: Pre-Pack HDRP Template Win trunk
     agent:
         type: Unity::VM
         image: package-ci/win10:stable
         flavor: b1.large
     commands:
-      -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-      -  upm-ci template pack --project-path com.unity.template-hd
+      -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
```

Comment:
arent we moving away from using pip for unity-downloader-cli?

Reply:
Thanks, I'll change that.